src/mailroom.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""
+The mailroom script allows someone to add donors to a database and generate a
+thank you letter. It also allows the user to view the list of donors in a table
+format, sorted by total amount donated. The data is persisted between
+application runs.
+"""
+
+import os
+import sys
+import io
+
+DEFAULT_DONORS = {
+    'Steven': [3, 6],
+    'David': [1, 5, 7],
+    'Zach': [5, 3, 6]
+}
+
+DEFAULT_LETTER = 'Thank you, {name} for your donation of {amount}.\n'
+
+
+def main_menu():    # pragma: no cover
+    """Main func that get executed when run in the CLI."""
+    try:
+        while True:
+            clear_screen()
+            print(welcome_message())
+            user_input = input("Selection: ")
+            if user_input not in ['1', '2', '3']:
+                continue
+            handle_input(user_input)()
+    except KeyboardInterrupt:
+        exit()
+
+
+def send_thanks():  # pragma: no cover
+    """Send thank-you note submenu."""
+    while True:
+        donor = input("Enter Donor name, 'list' or 'return': ")
```

Comment:
You've now stapled this test to user input, which makes it much harder to test.  

Can you figure out how to lift this requirement up and out of here, so that you can use only the one call to `input` on line 28 above?